README.md

```diff
@@ -40,6 +40,10 @@ Changes related to clients and arguments:
 
 See this commit diff as an example change: [python-client-migration/python_client_migration/main.py](https://github.com/aiven/opensearch-migration-examples/commit/f11b0e379dc63d9c023b62a032a72d9f1d4b9fc5)
 
+### Resources
+
+- [Compatibility](https://opensearch.org/docs/latest/clients/index/)
+- [Python](https://opensearch.org/docs/latest/clients/python)
```

Comment:
Maybe `Python Client` instead of `Python`?
(the Java resource says "Java high level client", but I don't think one needs the "high level")